apps/server/src/utils/parserFunctions.ts

```diff
@@ -255,7 +255,7 @@ export const parseCustomFields = (data: Partial<DatabaseModel>): CustomFields =>
 
   for (const fieldLabel in data.customFields) {
     const field = data.customFields[fieldLabel];
-    if (!field.label || !field.type || !field.colour) {
+    if (!('label' in field) || !('type' in field) || !('colour' in field)) {
```

Comment:
I wonder if we should be a bit more cautious in how we merge the fields after. is it worth checking the types?
type should always be "string"
colour should be of type string
label should be of type string and cannot be empty

like

```ts
    newCustomFields[key] = {
      type: field.type, 
      colour: field.colour,
      label: field.label,
    };
```